### DISCUSSION TITLE
[Vote ended on 2023-08-29] Remove ngine_io.exoscale from Ansible 10

### DISCUSSION THREAD

**Discussion by mariolenz** [ansible-community/community-topics#266] [Votes done]
`ngine_io.exoscale` looks unmaintained (#255).

Please vote with +1 to remove it from Ansible 10 or -1 to keep it.

**Comment by mariolenz** [comment]
-1

_edit:_ Changing to -1 now that [1.1.0](https://github.com/ngine-io/ansible-collection-exoscale/releases/tag/v1.1.0) has been released.

**Comment by resmo** [comment]
+/-1 maintainer of this collection

This collection is small and not as important to be included in the community package.

**Comment by Andersson007** [comment]
~~+1~~

UPDATE: -1 considering that it looks being maintained again

**Comment by markuman** [comment]
+1

**Comment by russoz** [comment]
0

(updated in 28/Aug)

**Comment by briantist** [comment]
+1

**Comment by felixfontein** [comment]
~~+1 - it doesn't really look unmaintained, but it doesn't really looks fully maintained and fully conforming to the requirements either.~~

Changing my vote to -1, as the situation got a lot better. I hope the next time the situation will change before we start voting :)

**Comment by gotmax23** [comment]
-1.

(See this comment's edit history. I previously voted +0, but there's been a new release, so I'm inclined to keep the collection in the ansible package.)

**Comment by jamescassell** [comment]
-1

**Comment by acozine** [comment]
-1 due to new release and recent fixes.
